src/aws-cpp-sdk-core/include/aws/core/http/HttpResponse.h

```diff
@@ -128,6 +128,71 @@ namespace Aws
                     return true;
                 default:
                     return false;
+                case HttpResponseCode::REQUEST_NOT_MADE:
```

Comment:
what is the actual warning? id rather not list all enum values if possible

Reply:
It looks like this:
```
warning: 15 enumeration values not explicitly handled in switch: 'AWS_MQTT5_PT_RESERVED', 'AWS_MQTT5_PT_CONNECT', 'AWS_MQTT5_PT_CONNACK'..
```